vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -48,7 +48,7 @@ type internal FSharpCheckerProvider
     ) =
     let checker = 
         lazy
-            let checker = FSharpChecker.Create()
+            let checker = FSharpChecker.Create(projectCacheSize = 200, keepAllBackgroundResolutions = false)
```

Comment:
@dsyme I meant if we revert that parameter change in the PR, can we use environment variable to override it?

Also, longer term, would making stuff that project relies on (like the same assembly metadata, which is probably retrieved for many projects in a solution) shared across the board have a significant impact on memory used?